mobile-widgets/qml/Export.qml

```diff
@@ -75,7 +75,6 @@ Kirigami.ScrollablePage {
 		onApply: {
 			if (selectedExport === ExportType.EX_DIVELOGS_DE) {
 				if (fieldUserID.text !== PrefCloudStorage.divelogde_user) {
```

Comment:
interestingly, our coding style would suggest to remove the curly braces here...

Reply:
not asking you to change this, just rambling

Reply:
I will bear that in mind, to be honest I tend to use {} too often (I have too often seen If if else go wrong :-)